applications/vanilla/modules/class.promotedcontentmodule.php

```diff
@@ -93,6 +96,10 @@ public function load($parameters = []) {
         }
     }
 
+    public function setShowIfNoResults($val) {
```

Comment:
Need a doc block here

Reply:
`ShowIfNoResults` is a public property in this class. A setter is kinda pointless. If you want to force a boolean (which is a good idea), I recommend lowering the visibility of this property down to private and making a getter method to fetch the value.